Volodymyr_Kuksa/3/brackets.py

```diff
@@ -0,0 +1,51 @@
+BRACKETS = ('(', ')', '{', '}', '[', ']')
+
+BRACKET_PAIRS = {
+    '(': ')',
+    '{': '}',
+    '[': ']'
+}
+
+OPENING_BRACKETS = tuple(BRACKET_PAIRS.keys())
+
+
+def is_bracket_pair(opening, closing):
+    """
+    Return True if opening and closing is a pair of matching brackets, False otherwise.
+
+    :param opening: an opening bracket.
+    :type opening: single-character str.
+
+    :param closing: a closing bracket.
+    :type closing: single-character str.
+
+    :return: True if opening and closing is a pair of matching brackets, False otherwise.
+    :rtype: bool.
+    """
+    return OPENING_BRACKETS.count(opening) and BRACKET_PAIRS[opening] == closing
```

Comment:
Why do you need to define one row function?

Reply:
You're right, that's overkill, but it used to be larger when I created it the first time.  
Will fix it with the next batch of tasks)